docs/developer-docs/latest/setup-deployment-guides/configurations.md

```diff
@@ -1575,3 +1575,108 @@ module.exports = () => ({
   },
 });
 ```
+
+### Role-Based Access Control <BronzeBadge withLinkIcon link="https://strapi.io/pricing" />
+
+Role-Based Access Control (RBAC) is an approach to restricting access to some users. In a Strapi application, users of the admin panel are administrators, and you can easily [configure their roles and permissions](/user-docs/latest/users-roles-permissions/configuring-administrator-roles.md). In addition to the 3 default roles (author, editor, and super admin), you can create custom conditions to match specific needs of roles and permissions management. This requires an EE license with a Bronze plan.
+
+::: warning 🚧 This API is considered unstable for now.
+<br>
+:::
+
+#### Defining a new condition
+
+Conditions are defined as an array of objects. Each condition object can have 4 possible properties:
+
+- `displayName`: the user name displayed in a friendly format, as showed in the admin panel
+- `name`: the condition name, kebab-cased
+- `plugin`: the plugin name, kebab-cased, if your condition is based on a plugin (e.g `content-manager`)
+- `handler`: a function to handle the user and possibly return a `condition object` (see [using the handler function](#using-the-handler-function))
+
+You declare and register conditions in your [`bootstrap`](/developer-docs/latest/setup-deployment-guides/configurations.md#bootstrap) file located at `./config/functions/bootstrap.js` (see [Adding conditions](#adding-conditions)).
+
+#### Using the handler function
+
+The condition `handler` is a synchronous or asynchronous function that:
+
+* receives the authenticated user making the request,
+* and returns `true`, `false`, or another condition object.
+
+Returning `true` means the condition will always match:
+
+```js
+  const condition = {
+    displayName: "Can drink",
+    name: "can-drink",
+    async handler(user) {
+      if (user.age > 21) return true;
+      return false;
+    },
+  };
+```
+
+Returning `false` means the condition will never match:
+
+  ```js
+  const condition = {
+    displayName: "Can drink",
+    name: "can-drink",
+    async handler(user) {
+      if (user.age < 21) return false;
+      return true;
+    },
+  };
+  ```
+
+Returning `true` or `false` is useful if you want to verify an external condition or a condition on the authenticated user only. For more granular control, you can return a condition object.
+
+A condition object is a matcher object used to verify conditions on the entities you `read`, `create`, `update` or `delete`.
+For instance, this custom condition uses an async handler that returns a condition object using the greater than (`$gt`) operator:
+
+```js
+  const condition = {
+    displayName: "price greater than 50",
+    name: "pricate-gt-50",
+    async handler(user) {
+      return { price: { $gt: 50 } };
+    },
+ };
+```
+
+We use [sift.js](https://github.com/crcn/sift.js) behind the scenes to match conditions. Here is the list of allowed operators:
```

Comment:
Avoid using "we": you should either replace with "the Strapi team" or rewrite the sentence differently.

Reply:
I would suggest replacing `we` with `Strapi` here instead of `the Strapi team` as it has nothing to do with the team in this case but at the application level.